config/jobs/kubernetes/sig-security/snyk-scan.yaml

```diff
@@ -0,0 +1,37 @@
+#  Periodic CI job for running snyk scans against k/k master
+# - It installs snyk CLI and requires 'snyk_token' secret available
+#   in test infra with key name 'SNYK_TOKEN'. This secret is used to
+#   populate env var 'SNYK_TOKEN', required for snyk CLI auth.
+# - Licenses and few false positive deps (eg version '0.0.0') are
+#   filtered from the snyk scan results and printed on stdout as well
+#   as "${ARTIFACTS}/snyk_results.json"
+periodics:
+  - interval: 6h
+    name: snyk-scan-kubernetes-master-periodical
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - image: golang
+        envFrom:
+        - secretRef:
+            # secret key should be defined as SNYK_TOKEN
+            name: snyk_token
```

Comment:
https://github.com/kubernetes/k8s.io/pull/2222 adds this to `k8s-infra-prow-build-trusted`